data/mods/Xedra_Evolved/mapgen/mutable/salamander_eruption.json

```diff
@@ -0,0 +1,725 @@
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "salamander_finale",
+    "weight": 100,
+    "object": {
+      "fill_ter": "t_obsidian_floor",
+      "rows": [
+        "####                ####",
+        "#@@@                @@@#",
+        "#@                  @###",
+        "#@                  @###",
+        "#@                  @###",
+        "#@                  @###",
+        "#@                  @###",
+        "##@@@               @###",
+        "####@@@             @###",
+        "######@@           @@@@#",
+        "#######@          @@####",
+        "#######@@@ @@@  @@@#####",
+        "#########@ @##@ @#######",
+        "#########@ @##@ @#######",
+        "#########@  @@@ @#######",
+        "#########@      @#######",
+        "#########@  @@  @#######",
+        "#########@   @  @#######",
+        "#########@  @@  @#######",
+        "#########@  @   @#######",
+        "#########@      @#######",
+        "#########@  @   @#######",
+        "#########@  @@  @#######",
+        "#########@   @  @#######"
+      ],
+      "terrain": { "#": "t_rock", "@": [ [ "t_rock", 4 ], [ "t_obsidian_floor", 1 ] ] },
+      "furniture": { " ": [ [ "f_null", 5 ], "f_rubble_rock" ] },
+      "items": { " ": { "item": "mine_equipment", "chance": 3 } },
+      "monster": { " ": { "monster": "mon_zombie_miner", "chance": 5 } },
+      "nested": { " ": { "chunks": [ [ "mine_minerals_equipment", 10 ], [ "null", 100 ] ] } }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "salamander_finale",
+    "weight": 50,
+    "object": {
+      "fill_ter": "t_obsidian_floor",
+      "rows": [
+        "####                ####",
+        "#@@@                @@@#",
+        "#@                  @###",
+        "#@        .....     @###",
+        "#@        ..o..     @###",
+        "#@        .....     @###",
+        "#@        ......    @###",
+        "##@@@     ......    @###",
+        "####@@@   ......    @###",
+        "######@@  ......   @@@@#",
+        "#######@  ......  @@####",
+        "#######@@@......@@@#####",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######",
+        "#########@......@#######"
+      ],
+      "terrain": { "#": "t_rock", "@": [ [ "t_rock", 4 ], [ "t_obsidian_floor", 1 ] ] },
+      "furniture": { " ": [ [ "f_null", 5 ], "f_rubble_rock" ] },
+      "items": { " ": { "item": "mine_equipment", "chance": 3 } },
+      "monster": { " ": { "monster": "mon_zombie_miner", "chance": 5 } },
+      "vehicles": { "o": { "vehicle": "trencher", "chance": 100, "fuel": 100, "status": 0 } },
```

Comment:
```suggestion
```

Reply:
Ick I got so distracted trying to get it to load after I came back to this I forgot to fix the dang content.  Thanks for reviewing @Night-Pryanik

Reply:
You're welcome!